### PR TITLE
Add libimagequant and pngquant

### PR DESCRIPTION
**Pull request by izahn**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by izahn** [comment]
I think this is ready for review!

**Comment by izahn** [comment]
@conda-forge/help-c-cpp it would be awesome if you can take a look at this. It is the last set of dependencies for ocrmypdf!

Thanks for all your time and efforts!

File: recipes/libimagequant/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    - {{ compiler('c') }}
+    - pkg-config
+    - make
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+
+test:
+  commands:
+    - bash -c "[ -f $PREFIX/lib/libimagequant.so ]"  # [linux]
+    - bash -c "[ -f $PREFIX/lib/libimagequant.0.dylib ]"  # [osx]
```
Comment (xhochy) [inline recipes/libimagequant/meta.yaml:29]:
```suggestion
    - test -f $PREFIX/lib/libimagequant.so  # [linux]
    - test -f $PREFIX/lib/libimagequant.0.dylib  # [osx]
```

File: recipes/libimagequant/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+about:
+  home: https://pngquant.org/
+  license: GPL-3.0-or-later
+  license_family: GPL2
```
Comment (xhochy) [inline recipes/libimagequant/meta.yaml:34]: We set that automatically when the license is SPDX compatible.
```suggestion
```

File: recipes/libimagequant/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+source:
+  url: https://github.com/ImageOptim/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 1f46deec3fa280335cf7f0be3ede6006bb420ba8159a03d1f6938e8d283d03ae
+
+build:
+  skip: true  # [win]
+  number: 0
+  rpaths:
+    - lib/
```
Comment (xhochy) [inline recipes/libimagequant/meta.yaml:16]: That shouldn't be needed.

File: recipes/pngquant/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+source:
+  url: https://github.com/kornelski/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 2d462ca5739ac0afe5a6851914a48ff9d06d4fbd3c4883e8308de5dcc6d444bc
+
+build:
+  skip: true  # [win]
+  number: 0
+  rpaths:
+    - lib/
```
Comment (xhochy) [inline recipes/pngquant/meta.yaml:16]:
```suggestion
  rpaths:
    - lib/
```

as above. If really needed, please add a comment why.

File: recipes/pngquant/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+    - libpng
+
+test:
+  commands:
+    - pngquant --version
+
+about:
+  home: https://pngquant.org/
+  license: GPL-3.0-or-later
+  license_family: GPL2
```
Comment (xhochy) [inline recipes/pngquant/meta.yaml:41]:
```suggestion
```

File: recipes/pngquant/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+  host:
+    - libimagequant
+    - lcms2
+    - libpng
+  run:
+    - libimagequant
+    - lcms2
+    - libpng
```
Comment (xhochy) [inline recipes/pngquant/meta.yaml:32]: Don't these libraries all have `run_exports`? Then you could omit them here.

**Review by xhochy — CHANGES_REQUESTED** [review]
`libimagequant` should have a `run_exports` specifying its ABI compatability: https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#export-runtime-requirements

**Review by xhochy — APPROVED** [review]

**Pull request merged**